client/src/app/core/repositories/agenda/agenda-item-repository.service.ts

```diff
@@ -128,13 +128,17 @@ export class AgendaItemRepositoryService extends MeetingModelBaseRepository<View
     }
 
     public async addItemToAgenda(contentObject: BaseViewModel & HasAgendaItem): Promise<Identifiable> {
-        return await this.create({ id: contentObject.id } as AgendaItem);
+        return await this.actions.sendRequest(ActionType.AGENDA_ITEM_CREATE, {
+            content_object_id: `${contentObject.collection}/${contentObject.id}`,
+            meeting_id: this.activeMeetingService.meetingId
+        });
     }
 
     public async removeFromAgenda(item: ViewAgendaItem): Promise<void> {
-        return await this.delete(item);
+        return await this.actions.sendRequest(ActionType.AGENDA_ITEM_DELETE, {
```

Comment:
A note for testing: deleting an agenda item can have two different effects: Deleting a topic must delete the topic, too. Deleting models of other collections just the ageda item has to be deleted.